### ISSUE TITLE
Source not found

### ISSUE THREAD

**Issue by Jiaion** [saltstack/salt#27458]
Hi all:
    Salt recent frequent occurrence, Source not found error

```
          ID: /etc/cron.d/salt
    Function: file.managed
      Result: False
     Comment: Source file salt://salt-client/template/salt.cron not found
     Started: 19:30:16.378430
    Duration: 2.481 ms
     Changes:
```
I'm sure that the file exists and master's

/var/log/salt/master 
```
2015-09-28 19:29:59,636 [salt.payload                             ][CRITICAL]  /usr/local/salt/packages/salt/payload.py:120 Could not deserialize msgpack message: This often happens when trying to read a file n
ot in binary mode.Please open an issue and include the following error: Unpack failed: error = 0
```

```
#salt --versions-report
           Salt: 2015.5.0
         Python: 2.6.6 (r266:84292, Sep 12 2011, 14:03:14)
         Jinja2: 2.8-dev
       M2Crypto: 0.21.1
 msgpack-python: 0.4.0
   msgpack-pure: Not Installed
       pycrypto: 2.6.1
        libnacl: Not Installed
         PyYAML: 3.10
          ioflo: Not Installed
          PyZMQ: 13.0.2
           RAET: Not Installed
            ZMQ: 3.2.2
           Mako: 0.9.0
        Tornado: Not Installed
        timelib: Not Installed
       dateutil: Not Installed
```

**Comment by jfindlay** [comment]
@Jiaion, thanks for the report.  Can you reproduce this on 2015.5.3, 2015.5.5, or 2015.8.0?

**Comment by Jiaion** [comment]
@jfindlay  
I use branch is 2015.5, commid 7a34c7742d59cc065fb856571b64d729a549fa8f 
My situation is the use of multiple Master's.

I found that often arise when more than one master issues such as previous https://github.com/saltstack/salt/issues/20240 , seems to have been resolved now.

**Comment by jfindlay** [comment]
@Jiaion, thanks for verifying.  I am going to close this issue as it seems to have already been fixed by #27234 as you have indicated.  If you encounter this problem again, we will reopen the issue.
